### PR TITLE
Added UWP samples

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
* NeonShooter
* Platformer2D
* SpaceWar  (Still WIP)

UWP Platforms added to these projects

### CODE REVIEW

File: NeonShooter/Game.cs [hunk 1]
```diff
@@ -46,10 +44,10 @@ public NeonShooterGame()
 			graphics = new GraphicsDeviceManager(this);
 			Content.RootDirectory = "Content";
 
-			graphics.PreferredBackBufferWidth = 800;
-			graphics.PreferredBackBufferHeight = 600;
+            graphics.PreferredBackBufferWidth = 1920;
```
Comment (CartBlanche) [inline NeonShooter/Game.cs:47]: Spacing issues?

**Comment by CartBlanche** [comment]
@SimonDarksideJ Shouldn't the Content.mgcb files be shared between platforms or does the UWP do something different to the other platforms with it's content?

**Comment by tomspilman** [comment]
@SimonDarksideJ - Am i waiting on any changes to this PR?

**Comment by SimonDarksideJ** [comment]
Not for the addition of the UWP platform. Will follow up with pr's to update the other platforms and deprecate others

**Comment by tomspilman** [comment]
@SimonDarksideJ @CartBlanche  Merged... didn't have a ton time to look at this closely, so sort of trusting this is good.

**Pull request merged**